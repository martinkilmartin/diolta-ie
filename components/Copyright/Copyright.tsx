type Props = {
  brand: string
}

const Copyright = ({ brand }: Props): JSX.Element => (
  <div className="flex items-center justify-center pt-10 font-light text-center sm:pt-12">
    {`Â© ${new Date().getFullYear()} ${brand}`}
  </div>
)

export default Copyright
