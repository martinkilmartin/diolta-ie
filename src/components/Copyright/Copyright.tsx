type Props = {
  brand: string
}

const Copyright = ({ brand }: Props): JSX.Element => (
  <p>{`Â© ${new Date().getFullYear()} ${brand}`}</p>
)

export default Copyright
