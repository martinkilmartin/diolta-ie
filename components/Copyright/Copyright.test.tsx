import renderer from 'react-test-renderer'
import Copyright from './Copyright'

test('Copyright displays current year and brand', () => {
  const brand = 'Acme Teo.'
  const copyrightRenderer = renderer.create(<Copyright brand={brand} />)
  const tree = copyrightRenderer.toJSON()
  expect(tree).toMatchSnapshot()
  const copyrightInstance = copyrightRenderer.root
  expect(
    copyrightInstance.findByProps({
      className:
        'flex items-center justify-center pt-10 font-light text-center sm:pt-12',
    }).children[0]
  ).toEqual(`Â© ${new Date().getFullYear()} ${brand}`)
})
