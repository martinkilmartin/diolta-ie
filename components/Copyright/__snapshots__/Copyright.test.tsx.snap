// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Copyright displays current year and brand 1`] = `
<div
  className="flex items-center justify-center pt-10 font-light text-center sm:pt-12"
>
  Â© 2021 Acme Teo.
</div>
`;
